packages/core/src/Repo.ts

```diff
@@ -452,3 +453,6 @@ export class Repo<T = any> extends EventEmitter {
     }
   }
 }
+
+// deep clone without Automerge metadata
+const getSnapshot = (o: any) => JSON.parse(JSON.stringify(o))
```

Comment:
Bug fix: When creating snapshots, we were making a shallow clone of the automerge document. (This worked fine in the grid example because properties on a grid row are all primitive values - nothing is nested beyond that first level.) 

I'll add a test for this before finalizing the PR.

Reply:
For performance, wouldn't it be better to make it overridable/configurable?

Reply:
A better solution would be to use a proper deepclone method - I'm sure Ramda has one. This would be just as performant as the spread when objects are shallow. 

I didn't go this route because I knew the serialization hack would strip the Automerge stuff, and I'd imagine something off the shelf would leave the properties with symbol names in there. I'll investigate.

Reply:
Ramda.clone doesn't copy properties with symbols for names, so it works for this situation. 3c17934d4c1e2dbfcbe41546c14c567039207d62